e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/dashboard-deals-filter-by-notice-type.spec.js

```diff
@@ -16,9 +16,7 @@ context('Dashboard Deals filters - filter by submissionType/noticeType', () => {
     cy.deleteGefApplications(ADMIN);
     cy.deleteDeals(ADMIN);
 
-    cy.insertOneDeal(BSS_DEAL_MIA, BANK1_MAKER1).then((deal) => {
-      ALL_DEALS.push(deal);
-    });
+    cy.createBssEwcsDeal({ readyForCheck: true, dealType: 'BSS/EWCS', facilityStage: 'Issued' });
```

Comment:
Can we use the `FACILITY_STAGE` constant instead of hard coding `Issued` please

Reply:
Replaced with `FACILITY_STAGE`